egs/hub4_spanish/s5/local/expand_graphemic_lexicon.sh

```diff
@@ -0,0 +1,64 @@
+#! /bin/bash
```

Comment:
is this file really needed in the recipe? Or is it just something you wrote for Chunxi?